onnxruntime/test/providers/cpu/tensor/pad_test.cc

```diff
@@ -25,17 +46,16 @@ static void RunTest(
   test1.Run();
 
   // ONNX domain opset-11
-  OpTester test2("Pad", 11);
-  if (mode != "constant") test2.AddAttribute("mode", mode);
-  test2.AddInput<float>("data", input_dims, input);
-  test2.AddInput<int64_t>("pads", {static_cast<int64_t>(pads.size())}, pads);
-  test2.AddInput<float>("value", {1}, {value});
-  test2.AddOutput<float>("output", output_dims, output);
-  // NGraph and TensorRT do not yet support opset-11 and builds break on this test, hence exclude the EP
-  test2.Run(OpTester::ExpectResult::kExpectSuccess, "", {kNGraphExecutionProvider, kTensorrtExecutionProvider});
+		RunTypedTest(input_dims,
```

Comment:
>		 [](start = 0, length = 2)

Please, remove tabs

Reply:
I helpful tip would be to enable .clang-format processing in your VS if you are using one. it'd automatically format your changes.

---
In reply to: [337285751](https://github.com/microsoft/onnxruntime/pull/2179#discussion_r337285751) [](ancestors = 337285751)

Reply:
Ran all the files through clang.